docs/staking/intro.rst

```diff
@@ -1,77 +1,164 @@
-Using Gaia
-==========
+Using The Staking Module
+========================
```

Comment:
this file is not included in the build. it's also sort of a duplicate with a few more files that I need to address